waspc/cli/Wasp/Cli/Terminal.hs

```diff
@@ -21,6 +22,11 @@ asWaspStartMessage = waspMessageWithEmoji "🐝"
 asWaspSuccessMessage :: String -> String
 asWaspSuccessMessage = waspMessageWithEmoji "✅"
 
+asWaspWarningMessage :: String -> String
+asWaspWarningMessage str = concat ["\n", waspMessageWithEmoji "👀" errorStr, "\n"]
```

Comment:
Hah ok these eyes are funny :D!

Normally some other emojis are used for warnings, e.g. yellow exclamation mark or smth like that, would it make sense to use that instead? Although eyes also make sense to me, however they are not conventional. Maybe they would be better fit for suggestions.

Reply:
Yeah, that was a placeholder I intended to change before taking this PR out of Draft. I tried a yellow warning icon at first but it didn't seem to render right in VS Code which made me nervous about using it: https://emojigraph.org/warning/

I plan to find something more akin to warning symbols that should have good unicode support. Maybe that one is good enough still.

Reply:
Here is how that one looked:
<img width="600" alt="Screen Shot 2022-01-20 at 10 20 18 AM" src="https://user-images.githubusercontent.com/523636/150368657-9fad2d98-ffd2-4c2c-bad1-8439decc1320.png">


And here is what I went with:
<img width="617" alt="Screen Shot 2022-01-20 at 10 24 30 AM" src="https://user-images.githubusercontent.com/523636/150368642-977ab76f-5470-4fbc-9590-3f74abb2bb3e.png">

Reply:
Ultimately changed color to red too, so it doesn't blend in:
<img width="623" alt="Screen Shot 2022-01-20 at 1 07 21 PM" src="https://user-images.githubusercontent.com/523636/150396626-04a9d8d8-84fc-42a6-a441-845b6fbe74d9.png">

Reply:
Final version came out nice with purple eyes:
<img width="1002" alt="Screen Shot 2022-01-21 at 9 00 15 AM" src="https://user-images.githubusercontent.com/523636/150541259-029fdc90-8843-4734-b7ad-3afcd9ab7ccb.png">